src/api/spec/features/beta/webui/requests_index_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'browser_helper'
+
+RSpec.describe 'Requests Index' do
+  let(:submitter) { create(:confirmed_user, :with_home, login: 'kugelblitz') }
+  let(:receiver) { create(:confirmed_user, :with_home, login: 'titan') }
+  let(:target_project) { receiver.home_project }
+  let(:target_package) { create(:package, name: 'goal', project_id: target_project.id) }
+  let(:source_project) { submitter.home_project }
+  let(:source_package) { create(:package, name: 'ball', project_id: source_project.id) }
+
+  let!(:bs_requests) do
+    create(:bs_request_with_submit_action,
+           description: 'Please take this',
+           creator: submitter,
+           target_package: target_package,
+           source_package: source_package)
+
+    create(:bs_request,
+           type: 'maintenance_release',
+           source_project: submitter.home_project,
+           creator: submitter)
+
+    create(:set_bugowner_request, creator: submitter)
+
+    create(:bs_request_with_maintenance_release_actions,
+           source_project: source_project,
+           state: :review,
+           creator: submitter)
+  end
+
+  context 'request index page' do
+    describe 'show all requests' do
+      before do
+        login submitter
+        Flipper.enable(:request_index)
+        visit requests_path(params: { involvement: 'all' })
+      end
+
+      it 'list all requests for the logged user' do
+        expect(page).to have_text(submitter.requests.first.bs_request_actions.first.class.sti_name.to_s.humanize.titleize)
+        expect(page).to have_text(submitter.requests.first.creator)
```

Comment:
All pages will have this...

Reply:
Why? Where?

Reply:
With the data you create for this spec, any page you will look at, as any user, will include `submitter.requests.first.creator`